core/base/libs/socrate/socrate/system.py

```diff
@@ -37,11 +40,74 @@ def set_env(required_secrets=[]):
     for secret in required_secrets:
         os.environ[f'{secret}_KEY'] = hmac.new(bytearray(secret_key, 'utf-8'), bytearray(secret, 'utf-8'), 'sha256').hexdigest()
 
-    return {
+    if not os.environ.get('SUBNET'):
+        os.environ['SUBNET'] = get_network_v4()
+
+    to_return={
```

Comment:
Can't you just move this below the try-except block?

Reply:
IMHO we want failures to be visible here... why would we silence them?

Reply:
I was just talking about the location. It shouldn't go _in_ the try/except block, but _after_ it.
Why store it in "to_return" and return it later when it can just be a `return { ... }` like before?

Reply:
Sure, that's d38ddfff